src/entities/SecurityToken/Issuance/Offerings.ts

```diff
@@ -136,24 +141,14 @@ export class Offerings extends SubModule {
   /**
    * Launch a Tiered STO
    *
-   * @param startDate date when the STO should start
-   * @param endDate date when the STO should end
-   * @param tiers tier information
```

Comment:
I would leave this as

`@param tiers - array that specifies how many tokens to sell at each tier (along with their price and potential discounts)`